### DISCUSSION TITLE
About dataset used by TexTeller V2/V3

### DISCUSSION THREAD

**Discussion by Syencil** [OleehyO/TexTeller#74] [General]
V2和V3用了更多的数据去训练，那么这个数据集和第一版中所用的550K cleaned数据集比较起来，除了数量更多，还有什么提升和改变呢？额外的数据是如何获取的，或者说新的数据集有计划开源分享吗？

**Comment by Jeffrey98-AI** [comment]
补充一个问题，关于中文、英文、公式混合文本，标注数据的规范是什么样呢

**Comment by OleehyO** [comment]
关于数据的各种细节之后会发篇paper详细讨论，发表之后新的数据集就会开源。
